src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -392,7 +388,7 @@ protected final boolean passesCallThreshold(final double conf) {
                 perAlleleQuals.add((int)Math.round(AFresult.log10ProbOnlyRefAlleleExists()*-10));
             }
 
-            attributes.put(GATKVCFConstants.AS_QUAL_KEY, perAlleleQuals.stream().mapToInt(q -> Math.round(q)).boxed().collect(Collectors.toList()));
+            attributes.put(GATKVCFConstants.AS_QUAL_KEY, perAlleleQuals.stream().mapToInt(Math::round).boxed().collect(Collectors.toList()));
```

Comment:
Missed this in the first round but `mapToInt(lambda).boxed()` ought to be equivalent to `map(lambda)`